### DISCUSSION TITLE
Advanced Menu in lxqt

### DISCUSSION THREAD

**Discussion by subhaniminhas** [lxqt/lxqt#2181] [Ideas]
I am the lead developer of PakOS (https://sourceforge.net/projects/pakos/), which is based on Debian. 
I love to use lxqt, as the only flavor of PakOS, but since there is no advance menu available in lxqt, so I have to use xfce menu instead. Other than that, all underlying technology is lxqt. 
If there was an advanced menu available in lxqt, like xfce whisker menu or Mint main menu, lxqt would have been perfect, lightweight as well as fully featured.

**Comment by tsujan** [comment]
The feature request for something like "whisker menu" was made a long time ago. No LXQt dev has been motivated to implement it because its functionalities are provided in other ways. But we accept patches; contributors are always welcome to make PRs.

**Reply by subhaniminhas** [reply]
If I am able to convince a few undergrad students to develope a whisker menu like alternative for lxqt, where they should start and what standard they should follow. Any help or links will be welcome. Thanks.

**Reply by tsujan** [reply]
The code of the main menu plugin itself can be considered a standard.

**Reply by YamiYukiSenpai** [reply]
I think this is it!
https://github.com/elviosak/plugin-wingmenu
![Screenshot_20221219_203143](https://user-images.githubusercontent.com/13153124/208561014-78f0e3f4-8f10-42fd-8a51-9bc73c239935.jpg)

**Reply by stefonarch** [reply]
Discussion is here https://github.com/lxqt/lxqt/discussions/1991#discussioncomment-712393 too
